src/styles/variables/_colors.scss

```diff
@@ -1,9 +1,109 @@
+// Black
+$off-black-9: rgba(0, 0, 0, 0.9);
+$off-black-73: rgba(0, 0, 0, 0.73);
+$off-black-5: rgba(0, 0, 0, 0.5);
+$off-black-3: rgba(0, 0, 0, 0.3);
 $black: #000000;
-$white: #ffffff;
-$background: #0a0a0a;
 
-$primary: #b14eff;
-$warn: #e2dc4f;
+// Blues
+$anakiwa: #a1d2ff;
+$anakiwa-darker: #88d7ff;
+$anakiwa-lighter: #9ed0ff;
+$baby-blue: #9dd0ff;
+$blumine-darker: rgba(30, 81, 128, 0.2);
+$doger-blue-lighter: #3ca1ff;
+$doger-blue-darker-2: rgba(60, 161, 255, 0.2);
+$doger-blue-darker-05: rgba(60, 161, 255, 0.05);
+$doger-blue-darker-08: rgba(60, 161, 255, 0.08);
```

Comment:
Similar to the previous comment - could be better to handle opacity variants when they're used rather than explicitly setting them in vars.